src/main/java/io/buildpacks/example/sample/PeerReview.java

```diff
@@ -1,7 +1,5 @@
-// PeerReview.java
```

Comment:
1. Add a class-level comment at the beginning of the PeerReview class to describe its purpose briefly.